CONTRIBUTING.md

```diff
@@ -1,203 +1,332 @@
 # Contributing to OpenSauced Intro
-Contributions are always welcome, no matter how large or small. Before contributing, please read the [code of conduct](https://docs.opensauced.pizza/contributing/code-of-conduct/) and follow the directions below:
+
+Contributions are always welcome, no matter how large or small. Before contributing, please read the [code of conduct](https://docs.opensauced.pizza/contributing/code-of-conduct/) and follow the directions in this guide.
 
 ## Recommended Communication Style
+
 1. Always leave screenshots for visual changes.
-2. Always leave a detailed description in the Pull Request. Leave nothing ambiguous for the reviewer(s).
+2. Always leave a detailed description in the pull request. Leave nothing ambiguous for the reviewer(s).
 3. Always review your code first. Be sure to run the project locally and test it before asking for a review.
 4. Always communicate in the GitHub repository. Whether it is in the issue or the pull request, keeping the lines of communication open and visible to everyone on the team helps everyone around you.
 
 ## Issues
 
-If you wish to work on an open issue, please comment on the issue with `.take` and it will be assigned to you. If an issue is not assigned, it is assumed to be open for anyone to work on. Please assign yourself to an issue before beginning work on it to avoid conflicts.
+You can create an issue if you have an idea to add a section or a chapter to this course. But maintainers need to triage the issue before you can work on it. So please tag `@open-sauced/community` in the comment after you raise the issue.
+
+If you wish to work on an open issue, please comment with `.take`, and it will be assigned to you. If an issue is not assigned, it is assumed to be available for anyone to work on. So, assign yourself to an issue before beginning work to avoid conflicts.
+
+When you contribute to the project for the first time, please consider checking the [bug](https://github.com/open-sauced/intro/issues?q=is%3Aissue+is%3Aopen+label%3A%22%F0%9F%90%9B+bug%22) or [good first issue](https://github.com/open-sauced/intro/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) labels.
 
-If you are contributing to the project for the first time, please consider checking the [bug](https://github.com/open-sauced/intro/issues?q=is%3Aissue+is%3Aopen+label%3A%22%F0%9F%90%9B+bug%22) or [good first issue](https://github.com/open-sauced/intro/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) labels.
+If you get stuck, you can always ask for help in the [Discord](https://discord.com/invite/U2peSNf23P) or GitHub Discussions.
```

Comment:
[suggestion] It'd be a good idea to mention the specific channel in our Discord so people can have an easier time navigating the server. Same thing for the GitHub Discussions(hyperlinking it to be specific). 
```suggestion
If you get stuck, you can always ask for help in the Contributions chat section of our [Discord](https://discord.com/invite/U2peSNf23P) community or GitHub Discussions.
```